python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -113,7 +113,8 @@ def __init__(self, module, device, graph_json_str, dump_root):
         self._dump_root = dump_root
         self._dump_path = None
         self._run_individual = module["run_individual"]
-        self._run_individual_node = module["run_individual_node"]
+        if module.type_key != "rpc":
```

Comment:
This is to fix a bug introduced in PR #11000. @AndrewZhaoLuo see if you can take a look? Basically when a `Module` is of type `rpc`, `module['run_individual_node']` will throw an error of not able to find it.

Reply:
It's not a bug. You might want to bump the version of RPC server and try again